app/utils/db.py

```diff
@@ -601,6 +601,14 @@ def _parse_aggregate():
         else:
             return _starts_with_dollar(unique)
 
+    def _parse_sort_field(group_dict, sort):
+        field = sort[0][0]
+        sort_order = sort[0][1]
+        if sort_order > 0:
+            group_dict['$group'][field] = {'$min': '$'+field}
+        elif sort[0][1] < 0:
```

Comment:
If `sort_order` can't be 0, then `else` is enough here.

Otherwise, use `sort_order` rather than `sort[0][1]` again.

Reply:
according to documentation just 1 and -1! Updated. :)